File: dixit-companion/src/main/java/fr/erban/dxitcompanion/db/game/GameEntity.java
package fr.erban.dxitcompanion.db.game;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class GameEntity {

    @PrimaryKey(autoGenerate = true)
    private int idGame;

    private int pointsToWin;

    private boolean finished;

    private String nameWinner;

    private int nbTurns;

    private String scoreSheet;
}

File: dixit-companion/src/main/java/fr/erban/dxitcompanion/db/game/GameEntity.kt
package fr.erban.dxitcompanion.db.game

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity
data class GameEntity(
    @PrimaryKey(autoGenerate = true)
    val idGame: Int = 0,
    val pointsToWin: Int,
    val finished: Boolean,
    val nameWinner: String?,
    val nbTurns: Int,
    val scoreSheet: String?,
    val startedAt: Long = 0L,
    val endedAt: Long = 0L
)
